bin/fixbase/gwfixbase.ml

```diff
@@ -197,6 +203,8 @@ let speclist =
   ; ("-person-key", Arg.Set key, " missing doc")
   ; ("-index", Arg.Set index, " rebuild index. It is automatically enable by any other option.")
   ; ("-invalid-utf8", Arg.Set invalid_utf8, " missing doc")
+  ; ("-key", Arg.Set key, " fix duplicate keys")
```

Comment:
should we remove the "-person-key" option that does the same and only keep "-key"?

Reply:
Probably yes if you are ok with this removal.

Reply:
Humm, I would prefer we keep "-person-key" but with updating it's doc, and remove "-key" so we don't have to change any script running fixbase, and don't have a duplicate parameter

Reply:
Thanks!!